firstmicroservice/src/test/java/com/stgcodes/validation/AddressValidatorTests.java

```diff
@@ -16,29 +12,37 @@
 public class AddressValidatorTests {
 
     private Address address;
-    private AddressValidator addressValidator;
+    private ValidatorTestUtils validatorTestUtils;
     private BindingResult bindingResult;
-    private ResourceBundleMessageSource messageSource;
     List<String> testCases;
     private Map<String, String> errors;
 
     @Before
     public void setUp() {
         address = new Address("5678 What St.", "Unit 8", "Atlanta", "Georgia", "78019");
 
-        addressValidator = new AddressValidator();
-
-        messageSource = new ResourceBundleMessageSource();
-        messageSource.setBasename("ValidationMessages");
+        validatorTestUtils = new ValidatorTestUtils(new AddressValidator(), address);
 
         testCases = new ArrayList<>();
         errors = new HashMap<>();
     }
 
+    @Test
+    public void testValidLineOneValues() {
+        testCases = Arrays.asList( "   1234 Davis St.     ", "   1\n8\n9 Lakewood Ave. ", "Driveoiqi398038y5utoiwa");
```

Comment:
Lets discuss this approach.....This can be more readable modifying the Address object used in setup and then validating that object. 
- The basic idea would be to setup a valid Address object
- change a value in the test method to what you're trying to validate
- assert the validator (via errors) is performing as expected